src/components/history-page/entry-menu/entry-menu.tsx

```diff
@@ -9,24 +9,28 @@ import { Dropdown } from 'react-bootstrap'
 import { Trans, useTranslation } from 'react-i18next'
 import { ForkAwesomeIcon } from '../../common/fork-awesome/fork-awesome-icon'
 import { ShowIf } from '../../common/show-if/show-if'
-import { HistoryEntryOrigin } from '../history-page'
 import { DeleteNoteItem } from './delete-note-item'
 import './entry-menu.scss'
 import { RemoveNoteEntryItem } from './remove-note-entry-item'
+import { HistoryEntryOrigin } from '../../../redux/history/types'
+import { useSelector } from 'react-redux'
+import { ApplicationState } from '../../../redux'
 
 export interface EntryMenuProps {
   id: string;
   title: string
-  location: HistoryEntryOrigin
+  origin: HistoryEntryOrigin
   isDark: boolean;
   onRemove: () => void
   onDelete: () => void
   className?: string
 }
 
-export const EntryMenu: React.FC<EntryMenuProps> = ({ id, title, location, isDark, onRemove, onDelete, className }) => {
+export const EntryMenu: React.FC<EntryMenuProps> = ({ id, title, origin, isDark, onRemove, onDelete, className }) => {
   useTranslation()
 
+  const userExists = useSelector((state: ApplicationState) => !!state.user)
```

Comment:
userExists sounds a bit like you did a check on the server. I guess you mean something like `userSet` or `userLoggedIn` ?

Reply:
I actually used this name, because some time ago [you created a PR](https://github.com/hedgedoc/react-client/commit/a41d3d151583397420ebeea0d660ae54b658417d#diff-39cdc22456af1e934bd6a404e1911d19557fff57da6ce4c354699b98d05f9f22R22) that changed such selector variables to `userExists`. Of course, I can rename that here, making it inconsistent with the rest of the app.

Reply:
Yes. But KNOW I think this name is not accurate :yum: 
Your decision.

Reply:
I'll file a separate PR later on for changing this name in all places (once this is merged sometime).